src/consumer/component/consumer-component.js

```diff
@@ -656,7 +734,41 @@ export default class Component {
 
     return this.dists;
   }
-
+  async pack({
+    scope,
+    directory,
+    writeBitDependencies,
+    createNpmLinkFiles,
+    override
+  }: {
+    scope: Scope,
+    directory: ?string,
```

Comment:
keep in mind that for the directory you put the ? before the string and in writeBitDependencies and createNpmLinkFiles you put it after the arg.
Make sure you use the one you want.